astro/src/content/docs/release-notes/index.mdx

```diff
@@ -38,6 +38,21 @@ import IssueResolvedVia from 'src/components/docs/release-notes/ReleaseNoteIssue
 
 Looking for release notes older than 1.23.0? Look in the [release notes archive](/docs/release-notes/archive). Looking to be [notified of new releases?](/docs/operate/roadmap/releases#release-notifications) <span class="not-prose no-underline"><a class="ml-2" href="/docs/releases.xml"><i class="fas fa-xs fa-rss text-orange-700 text-2xl" width="50px" /></a></span>
 
+<ReleaseNoteHeading version='1.53.3' releaseDate='October 22, 2024'/>
+
+### Fixed
+<Issue issue="2764">
+The CSRF token for federated login added in version `1.47.0` is not applied when all identity providers configured for an application use managed domains, and the OAuth authorize request for the application includes an `idp_hint` parameter.
```

Comment:
We should be more consistent with this messaging. 

For example, here it is:

> The CSRF token for federated login added in version `1.47.0` is not applied when all identity providers configured for an application use managed domains, and the OAuth authorize request for the application includes an `idp_hint` parameter.

And below in the known issues for `1.47.0` we say:

> The CSRF token used with federated login is not being applied when all configured IdPs for an application use managed domains and an `/oauth2/authorize` request for the application includes an `idp_hint` parameter.

So we are not being consistent with our usage of `/oauth2/authorize` vs `OAuth authorize` - which should be `OAuth2` if we are going to use that word at all. 

I don't know if how often we used federated login in our docs or release notes. 

Could we say something like:

> In order to better protect 3rd party logins via SAML v2, OpenID Connect, and other 3rd party identity providers, a CSRF (cross site request forgery) token was added in version `1.47.0`. This token was not being used when all identity providers configured for the requested `client_id` were also configured to use managed domains, and the authorize request also contained the `idp_hint` request parameter. In this specific configuration, because the token was not being utilized, the login workflow would fail with an error indicating ?

We should always aim to:
1. Tell the end user why they care
2. What specific configuration causes the issue that we are going to fix
3. What is the symptom they will observe if they have or may hit this condition

This should help the customer know if they need to upgrade or if they are likely or unlikely to hit this bug that we are fixing. And as much as possible we should use the same terminology between the copy in the known issue and the fixes section of the target release.